docs/directives/IMAGE.md

```diff
@@ -0,0 +1,31 @@
+# `doc-image`
+
+Binds an image to a html element. For `IMG` tag elements the image url is bound to the `src` attribute:
```

Comment:
What does this mean? If you want to use `doc-image` you need to set the `src` attribute? Or does that it mean the editor will set the `src` attribute for you?

Reply:
The editor sets it for the user. I just wanted to describe the behavior of the image directive. Should I leave it out, or keep it?